Formula/stockfish.rb

```diff
@@ -14,8 +14,14 @@ class Stockfish < Formula
   end
 
   def install
-    arch = if MacOS.version.requires_popcnt?
-      "x86-64-modern"
+    arch = if Hardware::CPU.type == :arm
+      "apple-silicon"
+    elsif Hardware::CPU.avx2?
```

Comment:
I don't think we want al these as it might make it harder for bottles to work. Keep it to just arm for now.